### ISSUE TITLE
SOPS_AGE_KEY don't accept space separated keys

### ISSUE THREAD

**Issue by airone-x1** [getsops/sops#1361]
SOPS_AGE_KEY environment variable accept mutiples keys but thoses keys must be separated by new line. As an environment variable it would be more convenient to accept also any "whitespace" separator.

It's OK with :

```shell
$ export SOPS_AGE_KEY=$(cat age1.key age2.key)
```

But it's not OK with

```shell
$ export SOAPS_AGE_KEY="AGE-SECRET-KEY-xxxxxxxxx  AGE-SECRET-KEY-yyyyyyyyy"
```

**Comment by hiddeco** [comment]
We use the official parser from age itself, and I do not think we would like to replace this for something custom:

https://github.com/FiloSottile/age/blob/main/parse.go#L23

**Comment by airone-x1** [comment]
Yes, but Age don't use environnent variables so their readers are thougth for input file.

`SOPS_AGE_KEY` is Sops specific. It would be more user friendly to enable space separated keys and convert this value internally before calling Age parser.

**Comment by martin617github** [comment]
If this works:
```
$ export SOPS_AGE_KEY=$(cat age1.key age2.key)
```
then this might work also:
```
$ export SOPS_AGE_KEY=$(echo "AGE-SECRET-KEY-xxxxxxxxx"; echo "AGE-SECRET-KEY-yyyyyyyyy")
```
I came to this thread 10 minutes ago and I am still in the middle of testing myself, but the first results look promising

**Comment by sebz3** [comment]
```bash
export SOPS_AGE_KEY=$(cat age1.key age2.key)
```
This will generate env with newline character between keys so 

```bash
export SOPS_AGE_KEY=$'AGE-SECRET-KEY-xxxxxxxxx\nAGE-SECRET-KEY-yyyyyyyyy'
```
should be enough

**Comment by felixfontein** [comment]
I created a PR for this: #2086.
